src/resources/log4j2.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
does it work? is your file located correctly?

Reply:
Yes, everything is fine